### ISSUE TITLE
Option / Future Option? - Change Layout and/or remove Spotify-tui banner

### ISSUE THREAD

**Issue by maglore9900** [Rigellute/spotify-tui#1098]
I love spotify-tui, but I am using it embedded in the desktop (via custom wm and such) and the screen real estate is dominated by the banner field.

Is it possible to change the layout, disable the banner, etc?

If not, then please consider this a future request.

Thank you

**Comment by sewnie** [comment]
you can try https://github.com/aome510/spotify-player

**Comment by maglore9900** [comment]
> you can try https://github.com/aome510/spotify-player

I like spotify-player too! I have some issues getting it to recognize a blue-tooth speaker. Its layout is good though. Theme'ing meets my requirements as well.

**Comment by sewnie** [comment]
make an issue on its repository not here

**Comment by maglore9900** [comment]
@apprehensions yeah, not worried about that. Here I am looking for an answer to my original question.

**Comment by sewnie** [comment]
> Is it possible to change the layout, disable the banner, etc?

no, and probably not any time soon unless the project gets maintained and isnt abandonware anymore. i have explicitly requested the author of spotify-player to integrate this specific feature into it myself.
